apps/workplace/src/tests/book/new-desk-flow/new-desk-flow-form.component.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { FormGroup } from '@angular/forms';
+import { MatBottomSheet } from '@angular/material/bottom-sheet';
+import { Router } from '@angular/router';
+import { createRoutingFactory, Spectator } from '@ngneat/spectator/jest';
+import { BookingFormService } from '@placeos/bookings';
+import { Building, OrganisationService } from '@placeos/organisation';
+import { NewDeskFlowFormComponent } from 'apps/workplace/src/app/book/new-desk-flow/new-desk-flow-form.component';
+import { NewDeskFormDetailsComponent } from 'apps/workplace/src/app/book/new-desk-flow/new-desk-form-details.component';
+import { MockComponent } from 'ng-mocks';
+import { BehaviorSubject, of } from 'rxjs';
+
+describe('NewDeskFlowFormComponent', () => {
+    let spectator: Spectator<NewDeskFlowFormComponent>;
+    const createComponent = createRoutingFactory({
```

Comment:
Minor note but I routing isn't needed here so use `createComponentFactory` instead.
In my snippets in VSCode for generating the boilerplate for tests I mistakenly used the `createRoutingFactory` instead `createComponentFactory`. Sorry for adding a bit of extra work on your end.

Reply:
Resolved as discussed